app/build.gradle

```diff
@@ -26,9 +26,10 @@ android {
         applicationId 'org.listenbrainz.android'
         minSdk 21
         targetSdk 33
+        versionCode 8
```

Comment:
Unrelated. Please remove.

Reply:
My IDE automatically adds this stuff idk what is up with it